contributors/devel/cherry-picks.md

```diff
@@ -1,12 +1,13 @@
 # Overview
 
-This document explains how cherry picks are managed on release
-branches within the Kubernetes projects. A common use case for this
+This document explains how cherrypicks are managed on release
+branches within the Kubernetes projects.
```

Comment:
Isn't this only for k/k i.e. not all Kubernetes projects?

Reply:
hah, I think that's someone's typo (or otherwise optimism re: other k/repos) and yes. good catch!